gemfiles/rails6.gemfile

```diff
@@ -8,8 +8,10 @@ gem 'eventmachine' if ENV['EM']
 gem "activesupport",     ">= 6.0.0"
 gem "activemodel",       ">= 6.0.0"
 
+gem 'generator_spec'
+
 group :development do
-  gem 'pry'
   gem 'awesome_print', :require => false
+  gem 'pry'
```

Comment:
No need to swap pry

Reply:
Mmm, don't know why is this here, I'll check why it is here (I'm not using it at all).

Reply:
As you can see [here](https://github.com/nviennot/nobrainer/blob/master/gemfiles/rails6.gemfile#L12) it was already here, I just sorted them.